### ISSUE TITLE
[HOLD for payment 2022-11-07] [$250] BUG: Dev Console error when edit message  reported by @aimane-chnaif 

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#12048]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Login with any account
2. Go to any chat
3. Long tap any message
4. Tap "Edit comment"

## Expected Result:
No console errors

## Actual Result:
Console error Warning: Failed prop type: The prop `isComposerFullSize` is marked as required in `Composer`, but its value is `undefined`.

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- iOS
- Android

**Version Number:** 1.2.18-2
**Reproducible in staging?:** Need reproduction
**Reproducible in production?:** Need reproduction
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 

https://user-images.githubusercontent.com/43996225/197043807-6ea961fb-4b3c-47e1-98fa-68f5d79a17d4.mov

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @aimane-chnaif 
**Slack conversation:** https://expensify.slack.com/archives/C01GTK53T8Q/p1666283934468759

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by mallenexpensify** [comment]
@lschurr  new BZ chore SO is [here](https://stackoverflow.com/c/expensify/questions/14418), we want to ensure BZ stays assigned to all issues so we can get to BugZero,

**Comment by lschurr** [comment]
Triaging this today.

**Comment by lschurr** [comment]
Tagging Eng since this is a dev issue that I can't reproduce.

**Comment by dangrous** [comment]
Am indeed seeing this on dev. Since its only a warning, though, I think we can ignore it for now as it's not user facing, so I'll close this. However, ccing @neil-marcellini since it looks like it's related to this PR - https://github.com/Expensify/App/pull/9031 - in case there's a reason you can see to update, or it's a super quick fix?

**Comment by mallenexpensify** [comment]
We are compensating for bug reports we fix for console errors but... I hadn't thought of examples where they're only on dev.  Addressed internally https://expensify.slack.com/archives/C02NK2DQWUX/p1666661515587739?thread_ts=1663583691.520459&cid=C02NK2DQWUX

**Comment by neil-marcellini** [comment]
Yes @aimane-chnaif has the solution. I think we could also make it optional and default to `false`.

**Comment by neil-marcellini** [comment]
We should keep this open and fix it even though it's just a prop type warning.

**Comment by lschurr** [comment]
Job posted:
https://www.upwork.com/ab/applicants/1584959643548332032/job-details
https://www.upwork.com/jobs/~01a688c0ae4629126f

**Comment by dangrous** [comment]
Yep, that second proposal sounds great to me as well! I'll assign and lets go ahead with a PR.

**Comment by dangrous** [comment]
@lschurr this is reviewed and approved. Do we need to do anything else on this issue/Upwork/etc. other than merge? This is my first one!

**Comment by lschurr** [comment]
Also my first one - I asked [here](https://expensify.slack.com/archives/C01SKUP7QR0/p1666802688236379) for clarification on next steps.

**Comment by dangrous** [comment]
also @aimane-chnaif just wanted to share this info below in case you weren't aware how things work re: upwork, etc.!

I know there's a lot of information in our contributing guidelines, so some points to take note of:
1. Once your PR is merged, you can be hired for another issue. Once you've completed a few issues, we may start hiring you for more than one issue at a time.
2. Once your PR is deployed to our staging servers, it will undergo quality assurance (QA) testing. If we find that this doesn't work as expected or causes a regression, you'll be responsible for fixing it. Typically we would revert this PR and give you another chance to create a similar PR without causing a regression. (I don't imagine this will happen with this PR, but it's something to be aware of)
3. Once your PR is deployed to production, we start a 7-day timer. After it has been on production for 7 days without causing any regressions, then we pay out the Upwork job.

So it might take a while before you're paid for your work, but we typically post multiple new jobs every day, so there's plenty of opportunity. I hope you've had a positive experience contributing to this repo!

**Comment by dangrous** [comment]
@aimane-chnaif awesome! Sounds great.

**Comment by lschurr** [comment]
Hi @aimane-chnaif - Could you please apply for the job in Upwork so that we can hire you? https://www.upwork.com/jobs/~01a688c0ae4629126f

**Comment by lschurr** [comment]
Hi @eVoloshchak - could you apply to the job in Upwork so that I can pay you for C+?

**Comment by lschurr** [comment]
Hey @aimane-chnaif - Could you accept the offer in Upwork and I'll get that paid!

**Comment by lschurr** [comment]
All set. This has been fixed and paid!
